api/Window.json

```diff
@@ -1619,11 +1619,11 @@
             },
             "firefox": {
               "version_added": true,
-              "notes": "Firefox has a <a href='https://bugzil.la/1386683'>bug</a> whereby single quotes contained in URLs are escaped when accessed via URL APIs. This has been fixed as of Firefox 57."
+              "notes": "Firefox has a <a href='https://bugzil.la/1386683'>bug 1386683</a> whereby single quotes contained in URLs are escaped when accessed via URL APIs. This has been fixed as of Firefox 57."
             },
```

Comment:
May as well simplify this note a little while we're here. What do you think of `"Before Firefox 57, single quotes contained in URLs are escaped when accessed via URL APIs. See <a href='https://bugzil.la/1386683'>bug 1386683</a>."`

Reply:
Fixed in 66b81343ed410a569a22d8c1f310533608272f89.